package.json

```diff
@@ -41,13 +42,29 @@
     "@turf/bbox-polygon": "^6.0.1",
     "@turf/boolean-contains": "^6.3.0",
     "@turf/boolean-intersects": "^6.0.2",
+    "@typescript-eslint/eslint-plugin": "^4.14.1",
+    "@typescript-eslint/parser": "^4.14.1",
```

Comment:
I would like to remove this `eslint-config-react-app` dependencies, but @AdriSolid already tried and it fails in case of not adding them.
What about the plans of moving to typescript :stuck_out_tongue: ?

Reply:
The reason is mainly to have the ESLint config aligned with the template. So `eslint-config-react-app` needs TS packages..
I agree moving to TS! :laughing:

Reply:
Ok, having this config synced with the one on the template is a priority.